src/Microsoft.Extensions.DependencyInjection/DefaultServiceProviderFactory.cs

```diff
@@ -4,14 +4,31 @@ namespace Microsoft.Extensions.DependencyInjection
 {
     public class DefaultServiceProviderFactory : IServiceProviderFactory<IServiceCollection>
     {
+        private readonly ServiceProviderOptions _options = ServiceProviderOptions.Default;
+
+        public DefaultServiceProviderFactory()
+        {
+
+        }
+
+        public DefaultServiceProviderFactory(ServiceProviderOptions options)
```

Comment:
We don't use optional parameters in public APIs. Versioning them requires thought (it's doable but we don't use them).